frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -143,8 +142,8 @@ const getPayloads = (stepName: string, state: WizardState, hasOCS: boolean, t: T
       const { apiGroup, apiVersion, kind } = OCSServiceModel;
       const systemKind = getStorageSystemKind({ apiGroup, apiVersion, kind });
       return [
-        createStorageClusterPayload(state),
         createSSPayload(systemKind, StorageClusterSystemName),
+        createStorageClusterPayload(state),
```

Comment:
I have one doubt... ultimately we are doing this `Promise.all([createSS, createSC]);`
It can happen sometime (not always) that SC gets created first then SS, right ? cause promises will not guaranteed resolve in synchronous order.

Reply:
Or it won't be an issue cause even though SS is not created we have already sent a request for it and operator knows that SS will get created soon... so will not create one by default when SC gets created ?

Reply:
Agree. I am working on that in #9538 . Here it would not make sense to do that fix with this.
Thanks for pointing.

Reply:
how does 9538 affects this?

Reply:
That will fix it.